samza-log4j/src/main/java/org/apache/samza/config/Log4jSystemConfig.java

```diff
@@ -78,7 +81,8 @@ public String getSerdeClass(String name) {
   }
 
   public String getStreamSerdeName(String systemName, String streamName) {
-    String streamSerdeNameConfig = String.format(StreamConfig.MSG_SERDE(), systemName, streamName);
-    return get(streamSerdeNameConfig, null);
+    StreamConfig streamConfig =  new StreamConfig(this);
+    scala.Option<String> option = streamConfig.getStreamMsgSerde(new SystemStream(systemName, streamName));
```

Comment:
qq: so it seems that we do not plan to create logic streamId for the logging topic? Can we make it clear and document how the logging topic is created s.t. it does not require  a logic streamId?